HACKING.adoc

```diff
@@ -224,6 +235,32 @@ found in link:INSTALL.adoc#bootstrap[INSTALL.adoc].
 
 ==== Github's CI: Travis and AppVeyor
 
```

Comment:
Maybe say something about Appveyor here (even if just one sentence, "to be documented later")?